### PR TITLE
feat: List component

### PR DESCRIPTION
**Pull request by ozairkhan2810**
Added list component
- Spaced list: space between list
- Stacked list: list stacked to each other
image
image

### CODE REVIEW

File: documentation/index.html [hunk 1]
```diff
@@ -324,6 +324,40 @@ <h6 class="heading-6">This is h6 heading</h6>
...
+        <h3>List</h3>
+        <p class="text-content">Spaced Lists</p>    
+        <div class="p-1 flex">
+          <ul class="spaced-list box-shadow ml-1 p-1 pl-2">
+            <li class="list-normal px-1">Item 1</li>
+            <li class="list-normal px-1">Item 2</li>
+            <li class="list-normal px-1">Item 3</li>
+          </ul>
+          <ul class="spaced-list box-shadow ml-1 p-1">
+            <li class="text-decoration-none px-1">Item 1</li>
```
Comment (Manju-sree) [inline documentation/index.html:338]: what is px-1 , can give better name here

**Review by Manju-sree — APPROVED** [review]
code looks fine, can work on giving better names to css classes

File: documentation/index.html (hunk 1 above)
Comment (ozairkhan2810) [reply documentation/index.html:338]: left right padding, padding on x axis

**Review by paarthbajaj — APPROVED** [review]
Saw all the commits. Looks grreat. You can merge.

**Pull request merged**